terms.html

```diff
@@ -18,7 +18,7 @@
   <dd>
 A set of one or more <a>claims</a> created by a single <a>issuer</a>,
 that is constructed for the purpose of transferring it to some other <a>entity</a>. 
-The <a>claims</a> in a <a>verifiable credential</a> can be about different <a>subjects</a>.
+The <a>claims</a> in a <a>verifiable credential</a> can be about different <a>resources</a>.
```

Comment:
this is just saying a credential can have multiple subjects?.. again, it seems like it might be colliding with some of the RDF assumptions

Reply:
A credential can have multiple claims, each of which has its own subject, thus implying that a creential can 'have' multiple subjects.

~~~ json
  "credentialSubject": [ {
    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
    "degree": {
      "type": "BachelorDegree",
      "name": "Bachelor of Science and Arts"
    },
    { "id": "did:example:abcdef1234567",
      "name": "Jane Doe"
    }
]

How does that collide with some of the RDF assumptions?

Reply:
[example](https://v.jsld.org/cJzm2pBicL2jYbBooAH7Vmz8gABtK1cbpe27Ce9ZPoXNzAsEvoa7EJH2NXeCv4PUhpn5jvjKU3ApF9KTyKUPNCpvAnTegQfdNXUywHi3b4ypUfUTHtowW4rphRTgmDT6v2cg6MS2MEKsF6qAnVyf92xdpevd27ESfPAFAiaTAda6p5HigVFv6PBoPqkjuiH6AvZQrQtZdviJzxf4aGzYviKR1GDBTHcA6gxvoPzkqpAqTerDMyxCMdRxgf59LkbTSSLAd3qhTTybiZgGCK4LtKSnJf7f9gS9bUuX46ns9uSJHUoGcz9czeDvvwipCoRBUnqhMErSGDCmtsey5WhFAWxSnX4Pt1mEdmhkNfFNLCvhfi867TKBJoXefgKqhGgdUbrY2PvGgHK61c4RnB2uK3Hzzez)

These are "claims" / "statements" / "rdf triples":

```
<did:example:c276e12ec21ebfeb1f712ebc6f1> <https://www.w3.org/ns/credentials/examples#name> "Morgan Doe" .
<did:example:c276e12ec21ebfeb1f712ebc6f1> <https://www.w3.org/ns/credentials/examples#spouse> "did:example:ebfeb1f712ebc6f1c276e12ec21" .
<did:example:ebfeb1f712ebc6f1c276e12ec21> <https://www.w3.org/ns/credentials/examples#name> "Jayden Doe" .
<did:example:ebfeb1f712ebc6f1c276e12ec21> <https://www.w3.org/ns/credentials/examples#spouse> "did:example:c276e12ec21ebfeb1f712ebc6f1" .
<http://university.example/credentials/3732> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <https://www.w3.org/2018/credentials#VerifiableCredential> .
<http://university.example/credentials/3732> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <https://www.w3.org/ns/credentials/examples#RelationshipCredential> .
<http://university.example/credentials/3732> <https://www.w3.org/2018/credentials#credentialSubject> <did:example:c276e12ec21ebfeb1f712ebc6f1> .
<http://university.example/credentials/3732> <https://www.w3.org/2018/credentials#credentialSubject> <did:example:ebfeb1f712ebc6f1c276e12ec21> .
<http://university.example/credentials/3732> <https://www.w3.org/2018/credentials#issuer> <https://example.com/issuer/123> .
<http://university.example/credentials/3732> <https://www.w3.org/2018/credentials#validFrom> "2010-01-01T00:00:00Z"^^<http://www.w3.org/2001/XMLSchema#dateTime> .

```

Reply:
I can see they are triples, and that they are very different from the JSON example I gave. Perhaps others, e.g., @msporny or others that have better knowledge about this than I do, can shed some light here.